pom.xml

```diff
@@ -31,8 +31,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven-version>3.2.1</maven-version>
-        <bt.java.source>1.8</bt.java.source>
-        <bt.java.target>1.8</bt.java.target>
+        <bt.java.source>1.7</bt.java.source>
+        <bt.java.target>1.7</bt.java.target>
```

Comment:
If this is required, we need coordination as many components only support 1.8 and 1.8 syntax is all over the code. Given that we are planning to sunset 1.7 support this quarter, how about leave the default to 1.8 and ask those components that have to support 1.7 to override in their poms?

Reply:
This is just to compile the plugin, not for other projects. Right now, I can't use the plugin in nexus from rex-je which is a Java7 project, unless I modify the source and compile it locally. We can have Java 7 plugins in our toolchain right? (I'm sure maven plugins are compiled mostly Java7 if not Java6.)